wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/authentication/LoginWithGoogleScreen.kt

```diff
@@ -29,7 +35,13 @@ fun LoginWithGoogleScreen(
             Timber.i("Google sign in cancelled")
             onAuthCanceled()
         },
-        failedContent = { AuthErrorScreen() },
+        failedContent = {
+            if(!Network.isConnected(context)){
+                ErrorScreen(stringResource(LR.string.log_in_no_network))
+            }else {
+                AuthErrorScreen()
+            }
```

Comment:
Instead of using different error screens here, what would you think about always using the `ErrorScreen` you created and instead just having the text vary?

``` kotlin
import com.google.android.horologist.auth.composables.R as HR

//...

failedContent = {
    val message = if (Network.isConnected(context)) {
        HR.string.horologist_auth_error_message
    } else {
        LR.string.log_in_no_network
    }
    ErrorScreen(stringResource(message))
},
```

Reply:
yes, I think that would be an optimal solution for maintaining consistency throughout the app.